StarkInfra/StarkInfra/IssuingRule/IssuingRule.cs

```diff
@@ -18,9 +18,9 @@ namespace StarkInfra
     ///     <item>ID [string, default null]: unique id returned when an IssuingRule is created, used to update a specific IssuingRule. ex: "5656565656565656"
     ///     <item>Interval [string, default "lifetime"]: interval after which the rule amount counter will be reset to 0. ex: "instant", "day", "week", "month", "year" or "lifetime"</item>
     ///     <item>CurrencyCode [string, default "BRL"]: code of the currency that the rule amount refers to. ex: "BRL" or "USD"</item>
-    ///     <item>Categories [list of strings, default []]: merchant categories accepted by the rule. ex: new List<string>{ "eatingPlacesRestaurants", "travelAgenciesTourOperators" }</item>
-    ///     <item>Countries [list of strings, default []]: countries accepted by the rule. ex: new List<string>{ "BRA", "USA" }</item>
-    ///     <item>Methods [list of strings, default []]: card purchase methods accepted by the rule. ex: new List<string>{ "chip", "token", "server", "manual", "magstripe", "contactless" }</item>
+    ///     <item>Categories [list of strings, default null]: merchant categories accepted by the rule. ex: new List<string>{ "eatingPlacesRestaurants", "travelAgenciesTourOperators" }</item>
+    ///     <item>Countries [list of strings, default null]: countries accepted by the rule. ex: new List<string>{ "BRA", "USA" }</item>
+    ///     <item>Methods [list of strings, default null]: card purchase methods accepted by the rule. ex: new List<string>{ "chip", "token", "server", "manual", "magstripe", "contactless" }</item>
```

Comment:
list of MerchantCategories, list of MerchantCountries e list of CardMethods. Precisa mudar os exemplos também